package-lock.json

```diff
@@ -0,0 +1,8015 @@
+{
```

Comment:
We don't include the `package-lock.json` since we only cover development dependencies now.